app/views/admin/users.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render "subnavbar" %>
```

Comment:
There is lots of `<%= render "subnavbar" %>` in here. Please refactor to have a separate layout for admin users.